.travis.yml

```diff
@@ -8,6 +8,7 @@ env:
   - TEST_SUBTASK=default
   - TEST_SUBTASK=shapes
   - TEST_SUBTASK=pkiss
+  - TEST_SUBTASK=unambig
```

Comment:
you are now forcing Travis to execute the script section four times (good) ....
... but in the script section itself, the if clause can only be satisfied once, namely if TEST_SUBTASK is "unambig". For the other three executions, nothing is done in the script section, which at least cannot fail ;-)

So save some compute time, I would comment out the other three tasks for now